include/deal.II/base/mpi.h

```diff
@@ -708,6 +708,44 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
+     * Given a partitioned index set space, compute the owning MPI process rank
+     * of each element of a second index set according to the partitioned index
+     * set. A natural usage of this function is to compute for each ghosted
+     * degree of freedom the MPI rank of the process owning that index.
```

Comment:
I stumbled over this because I though "But we know which rank a ghost DoF belongs to based on the subdomain id of the cell it is on". But this heuristic fails for DoFs on interfaces between ghost cells with different subdomain_ids, or between a ghost cell and an artificial cell." If you feel like it, paraphrase these two sentences and put them into parentheses after your comment to help readers not fall into the same thought as I just did.